docs/sprints/current/SPRINT_18_FOUNDATION_CLEANUP.md

```
@@ -0,0 +1,244 @@
+## 📋 Implementation Details
+
+### 1. Fleet Analyzer Cleanup
+**File**: `lib/eve_dmv/contexts/fleet_operations/domain/fleet_analyzer.ex`
+
+**Current (Bad)**:
+```elixir
+defp get_ship_class(ship_type_id) do
+  case rem(ship_type_id, 10) do
+    1 -> :frigate
+    2 -> :destroyer
+    3 -> :cruiser
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)